ingress/controllers/nginx/nginx.tmpl

```diff
@@ -153,8 +153,8 @@ http {
 
     {{ range $server := .servers }}
     server {
-        listen 80;
-        {{ if $server.SSL }}listen 443 ssl http2;
+        listen 80{{ if $cfg.useProxyProtocol }} proxy_protocol{{ end }};
+        {{ if $server.SSL }}listen 443{{ if $cfg.useProxyProtocol }} proxy_protocol{{ end }} ssl http2;
```

Comment:
Can i trouble you to add a proxy protocol example to the docs? I've actually heard others asking for the same and explaining it everytime is burdensome. If you have the time, I suggest a section that describes eg: start haproxy by creating this pod, now start the nginx controller behind haproxy, now do this to see src etc ...doesn't need to be very elaborate, just demonstrate a situation where user doesn't get src, and they does get it.